CMakeLists.txt

```diff
@@ -145,6 +151,8 @@ include (${SHARED_DIR}/utils/uncommon/shared_uncommon.cmake)
 set (THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+add_definitions(-DCOMPILING_WASM_RUNTIME_API=1)
```

Comment:
What I meant is to add control here, how about:
```cmake
if (MSVC)
  add_definitions(-DCOMPILING_WASM_RUNTIME_API=1)
endif ()
```

Reply:
oops my bad -- added